libs/parent-dashboard/feature/src/lib/parent-dashboard-feature.module.ts

```diff
@@ -3,9 +3,10 @@ import { CommonModule } from '@angular/common';
 import { ParentDashboardComponent } from './parent-dashboard';
 import { ParentDashboardRoutingModule } from './parent-dashboard-routing.module';
 import { IonicModule } from '@ionic/angular';
+import { NavbarModule } from '@the-au-pair/shared/components/navbar';
 
 @NgModule({
-  imports: [CommonModule, ParentDashboardRoutingModule, IonicModule],
+  imports: [CommonModule, ParentDashboardRoutingModule, IonicModule,NavbarModule],
```

Comment:
The same space inconsistency in all of them

Reply:
Updated